tools/kompos/src/visualizations.ts

```diff
@@ -31,9 +31,11 @@ function hashAtInc(hash, idx: string, inc: number) {
   }
 }
 
+var actorsPerType = {};
```

Comment:
hm, ok, here is the point where I would like to ask you to do a more thorough refactoring please.
Now we get too much state just lying around in globals.
Would be better to have a proper class for this code. Otherwise, things will become more ugly soon.